dotCMS/src/main/java/com/dotmarketing/portlets/folders/business/FolderAPIImpl.java

```diff
@@ -50,613 +50,620 @@
 import static com.dotmarketing.business.APILocator.getPermissionAPI;
 
 public class FolderAPIImpl implements FolderAPI  {
+    
+    public static final String ROOT_PATH = "/";
+    public static final String SYSTEM_FOLDER_PATH = "/System folder";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Refactor your code to get this URI from a customizable parameter. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1075)